src/main/java/com/teragrep/cfe18/handlers/CaptureMetaController.java

```diff
@@ -201,4 +202,34 @@ public ResponseEntity<String> removeCaptureMeta(@PathVariable("capture_id") int
     }
 
 
+    // Key value fetch
+    @RequestMapping(path="/{key}/{value}",method= RequestMethod.GET, produces="application/json")
+    @Operation(summary = "Fetch capture definitions by key and value")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Found the capture definitions",
+                    content = {@Content(mediaType = "application/json",
+                            schema = @Schema(implementation = CaptureMeta.class))}),
+            @ApiResponse(responseCode = "400", description = "Capture meta key or value does not exist",
```

Comment:
shouldn't it be the legendary 404? if similar use for 400 is elsewhere in the code base, then please create a fix issue.